### DISCUSSION TITLE
Finally migrated my site from Neuron!

### DISCUSSION THREAD

**Discussion by fiatjaf** [srid/emanote#408] [Show and tell]
Here's the old one: https://fiatjaf.com/neuron-site/
And the new one: https://fiatjaf.com/

@srid if you want to add this to the list of example sites it will be a pleasure.

I still have to catch up with the features and Vim plugins so I can get back to writing more, since I have almost completely stopped in the last couple of years, but I guess this was a good start.

**Comment by srid** [comment]
Great! I've added it to https://emanote.srid.ca/examples

By the way, you may want to replace `[[[...]]]` style links with `[[..]]#`. The former was deprecated, and emanote doesn't support it anymore. The latter may be extended in future: https://github.com/srid/emanote/discussions/358#discussioncomment-4635092

In regards to vim plugin, perhaps this is a good starting point: https://github.com/oberblastmeister/neuron.nvim/issues/45#issuecomment-1245622659

**Reply by fiatjaf** [reply]
Done that now, thanks.

**Comment by fiatjaf** [comment]
I've made the source public on GitHub now, if that's interesting: https://github.com/fiatjaf/z
